skyverge/wc-plugin-framework
# Payment form JS is never loaded at checkout if a $0 order has a total added later [6]

## Summary

This PR solves the issue where payment fields won't initialize when the customer starts the checkout with $0 order.

### Story: [MWC-31](https://jira.godaddy.com/browse/MWC-31)

## QA

1. Configure the Cybersource gateway and confirm the payment form JS is functional (any other direct gateway you have running works as well)
2. Add a shippable product that costs $0
3. Add a shipping zone that applies to a specific region or zip code, and a shipping method to it that costs $1
4. Ensure the fallback "Locations not covered by your other zones" zone has no methods enabled
5. Visit the product in an incognito window
6. Add to cart and visit checkout
    - The total is $0 (no shipping chosen yet)
    - No payment form is present
7. Fill in your billing details so that it matches your shipping zone and shipping becomes available
    - The order is > $0
    - [ ] The payment form is loaded, and once the method selected the JS get applied (auto-formatting, etc...)
